test/elixir_internal_certificate/account/accounts_test.exs

```diff
@@ -0,0 +1,204 @@
+defmodule ElixirInternalCertificate.Account.AccountsTest do
+  use ElixirInternalCertificate.DataCase
+
+  alias ElixirInternalCertificate.Account.Accounts
+  alias ElixirInternalCertificate.Account.Schemas.{User, UserToken}
+
+  describe "get_user_by_email/1" do
+    setup do
+      %{user: insert(:user)}
+    end
+
+    test "when the email does not exist, does not return the user" do
+      assert Accounts.get_user_by_email("unknown@example.com") == nil
```

Comment:
in this case, do we really `create a user in the database through the hook above` that could make the test suite a bit slowly 🤞